be/src/exec/pipeline/pipeline_driver.h

```diff
@@ -99,6 +99,17 @@ class DriverAcct {
     int64_t accumulated_chunk_moved{0};
 };
 
+// OperatorExecState is used to guarantee that some hooks of operator
+// is called exactly one time during whole operator
+enum OperatorStage {
+    INIT = 0,
```

Comment:
Do we really need the `INIT` stage?

Reply:
Not currently used, just for a complete definition